app/models/state_file_az_intake.rb

```diff
@@ -119,6 +119,8 @@ class StateFileAzIntake < StateFileBaseIntake
 
   validates :made_az321_contributions, inclusion: { in: ["yes", "no"]}, on: :az321_form_create
   validates :az321_contributions, length: { maximum: 10 }
+
+  validates :az322_contributions, length: { maximum: 10 }, on: :az322
```

Comment:
why is the implementation for this different from the one above `:az321_contributions`?

Reply:
The az322 (and originally, az321) contributions both were implemented with a boolean value on the contributions model which should have been on the intake ("made_contribution"), largely because of misleading mocks which referenced another similar page without noting the important difference. The az_321 contributions model was then refactored to move that boolean up to the intake, but (I learned while doing this story), that refactor never happened for the az322 contributions model. I'll go more into this at computer gardening next week, but let me know if that's not clear.